build.sbt

```diff
@@ -162,6 +162,9 @@ lazy val mongodb = alpakkaProject("mongodb", "mongodb", Dependencies.MongoDb)
 lazy val mqtt = alpakkaProject("mqtt", "mqtt", Dependencies.Mqtt)
 
 lazy val mqttStreaming = alpakkaProject("mqtt-streaming", "mqttStreaming", Dependencies.MqttStreaming)
+lazy val mqttStreamingBench = alpakkaProject("mqtt-streaming-bench", "mqttStreamingBench", Seq.empty)
+  .enablePlugins(JmhPlugin)
+  .dependsOn(mqtt, mqttStreaming)
```

Comment:
I think it would be better to move jmh tests to their corresponding (`mqtt` and `mqttStreaming`) projects.

Reply:
I believe that they must be in their own sub project given code generation. Do I have that wrong? I recall reading something to that effect from the sbt-jmh doc.

Reply:
Ahh, you are right. sbt-jmh does indeed say that. Lets leave it as it is now then. Probably in the future we will want one project for all benchmarks. Otherwise it will end up with a cambrian explosion of benchmark projects.